lib/blog/http_camera_connection.md

```diff
@@ -7,4 +7,92 @@ hero_image_credit_text: "Oppkey Developer Intern"
 hero_image_credit_link: "https://oppkey.com"
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tempus urna et pharetra pharetra massa massa ultricies mi quis. Gravida rutrum quisque non tellus orci ac auctor augue mauris. Vitae turpis massa sed elementum tempus. Non blandit massa enim nec dui nunc mattis enim ut. Malesuada fames ac turpis egestas maecenas pharetra convallis posuere morbi. Eget duis at tellus at. Nibh nisl condimentum id venenatis a condimentum vitae sapien pellentesque. Libero nunc consequat interdum varius sit. Ultrices eros in cursus turpis massa tincidunt dui ut. Ornare quam viverra orci sagittis eu volutpat odio facilisis. Dignissim cras tincidunt lobortis feugiat vivamus at augue. Hendrerit dolor magna eget est lorem ipsum dolor sit amet. Amet consectetur adipiscing elit ut aliquam purus sit amet luctus. Nisi quis eleifend quam adipiscing vitae proin. Iaculis urna id volutpat lacus laoreet non curabitur. Aliquam malesuada bibendum arcu vitae. At lectus urna duis convallis convallis.
+# Sending an API command to the camera
+
+This tutorial is the first in a series aimed at helping developers learn how to work with the RICOH THETA X camera. It focuses on using the API to build a mobile app. The examples provided use Flutter to build mobile apps. Source code is downloadable and open to pull requests on GitHub.
+
+# Main Resources
+
+* [RICOH THETA API](https://api.ricoh/docs/theta-web-api-v2.1/)
+* [theta360.guide developers community](https://www2.theta360.guide/)
```

Comment:
use theta360.guide instead of www2.theta360.guide as we may change the www2 prefix at some point and then the link will be broken

Reply:
Changed